File: Tarea4/ejercicio1.js
// recorrido de arreglo por columnas

let matriz = [
    [1, 2, 3],
    [4, 5, 6],
    [7, 8, 9]
];

let filas = matriz.length;
let columnas = matriz[0].length;

for (let j = 0; j < columnas; j++) {
    for (let i = 0; i < filas; i++) {
        console.log(matriz[i][j]);
    }
    console.log("---");
}

File: Tarea4/ejercicio1.cpp
// recorrido de arreglo por columnas

#include <iostream>
using namespace std;

int main() {
    int matriz[3][3] = {
        {1, 2, 3},
        {4, 5, 6},
        {7, 8, 9}
    };

    int filas = 3, columnas = 3;

    for (int j = 0; j < columnas; j++) {
        for (int i = 0; i < filas; i++) {
            cout << matriz[i][j] << " ";
        }
        cout << endl;
    }

    return o;
}